impeller/renderer/backend/vulkan/context_vk.cc

```diff
@@ -80,15 +98,17 @@ static bool HasRequiredProperties(const vk::PhysicalDevice& device) {
 
 static bool IsPhysicalDeviceCompatible(const vk::PhysicalDevice& device) {
   if (!HasRequiredQueues(device)) {
-    VALIDATION_LOG << "Device doesn't have required queues.";
+    FML_LOG(ERROR) << "Device doesn't have required queues.";
     return false;
   }
-  if (!HasRequiredExtensions(device)) {
-    VALIDATION_LOG << "Device doesn't have required extensions.";
+  auto missing_exts = HasRequiredExtensions(device);
+  if (!missing_exts.empty()) {
+    FML_LOG(ERROR) << "Device doesn't have required extensions: "
+                   << fml::Join(missing_exts, ", ");
     return false;
   }
   if (!HasRequiredProperties(device)) {
-    VALIDATION_LOG << "Device doesn't have required properties.";
+    FML_LOG(ERROR) << "Device doesn't have required properties.";
```

Comment:
these are no longer validation logs as there can be multiple physical devices and we don't need to fail early if one of them isn't valid.

Reply:
Perhaps just remove the log line entirely?